README.md

```diff
@@ -1,110 +1,178 @@
-# ojd_daps_skills
+# Skills Extractor
 
```

Comment:
we need a picture! or at least maybe a visual of a job advert text with the bits of it getting highlighted and then mapped